javascript-and-json/main.js

```diff
@@ -0,0 +1,18 @@
+var books = [{ isbn: '978-0156012195', title: 'The Little Prince', author: 'Antoine de Saint-Exupéry' },
+  { isbn: '978-0060887964', title: 'The Alchemist', author: 'Paulo Coelho' },
+  { isbn: '978-0451524935', title: '1984', author: 'George Orwell' }];
+
+console.log(books, typeof books);
+
+console.log(JSON.stringify(books), typeof books);
+
+var student = '{' +
+  '"Number id" : "299513512",' +
+  '"String Name"  : "Henry"' +
```

Comment:
The name of the properties should just be `id` and `name`, the other portion was just telling you what the data type should be. Also, I would prefer if you wrote this all as one string rather than concatenating it together. There are not values being injected into the string, so it can just be written as one string.